pkg/meta/model/job_args_test.go

```diff
@@ -226,3 +227,75 @@ func TestTruncateTableArgs(t *testing.T) {
 		require.Equal(t, []int64{5, 6}, args.OldPartitionIDs)
 	}
 }
+
+func TestRenameTableArgs(t *testing.T) {
+	inArgs := &RenameTableArgs{
+		OldSchemaID:   9527,
+		OldSchemaName: model.NewCIStr("old_schema_name"),
+		NewTableName:  model.NewCIStr("new_table_name"),
+	}
+
+	jobvers := []JobVersion{JobVersion1, JobVersion2}
+	for _, jobver := range jobvers {
+		job := &Job{
+			Version: jobver,
+			Type:    ActionRenameTable,
+		}
+		job.FillArgs(inArgs)
+
+		// encode job
+		_, err := job.Encode(true)
+		require.NoError(t, err)
+		job.Args = nil
+
+		// get the args after decode
+		args, err := GetRenameTableArgs(job)
+		require.NoError(t, err)
+		require.Equal(t, args, inArgs)
```

Comment:
```suggestion
		require.Equal(t, inArgs, args)
```
the first param is `expected`

Reply:
done